Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
Would it be more modern to use [just] and `justfile`?

[just]: https://just.systems/man/en/

Reply:
Ah, but Sphinx has `Makefile` support out of the box, and probably no support for `just`...

Reply:
I'll be honest this is the first I've heard of just and justfile, I do like the support out of the box and know how this works with make if you're good with that